docs/blog/2020-06-18-Gatsby-cli-survey-2/index.md

```diff
@@ -71,6 +71,6 @@ Requests for incremental builds and complaints about slow builds have decreased
 
 Thank you again to all who have responded to the survey so far; your responses help guide the focus of the community to answer real-world needs. If you haven't filled out the survey yet, you can run `gatsby feedback` at any time after upgrading your version of `gatsby` in a project's `package.json` file, and then rerunning dependency and install commands. We would love to hear from you!
 
-Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most, and then report results. (If you'd like to opt out, visit [https://www.gatsbyjs.org/docs/CLI-feedback/#how-to-opt-out](docs/telemetry/#how-to-opt-out/)).
+Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most, and then report results. (If you'd like to opt out, visit [/docs/CLI-feedback/#how-to-opt-out](docs/telemetry/#how-to-opt-out/)).
```

Comment:
This is actually the link text and doesn't need to be changed.

I think we can just avoid having the URL in the blog post anyways:

```suggestion
Otherwise, wait to see the survey appear in a CLI near you! The plan is to do this every 3 months, at the most, and then report results. (If you'd like to opt out, see the [Telemetry Docs](docs/telemetry/#how-to-opt-out/)).
```

Reply:
Oh yes. I miss that this was a title

But then the link is still a 404 because of missing `/` before `doc`